FABulous/fabric_generator/file_parser.py

```diff
@@ -684,6 +684,12 @@ def parseFile(
                 bits = details.get("bits", [])
                 filtered_ports[port_name] = (direction, bits)
 
+        if individually_declared:
+            logger.warning(
+                f"Ports in {filename} have been individually declared rather than vector."
```

Comment:
```suggestion
                f"Ports in {filename} have been individually declared rather than as a vector."
```
Reads a bit nicer IMHO.